markets/perps-market/contracts/modules/GlobalPerpsMarketModule.sol

```diff
@@ -83,6 +84,32 @@ contract GlobalPerpsMarketModule is IGlobalPerpsMarketModule {
         return GlobalPerpsMarketConfiguration.load().synthDeductionPriority;
     }
 
+    /**
+     * @inheritdoc IGlobalPerpsMarketModule
+     */
+    function setCollateralLiquidateRewardRatio(
```

Comment:
maybe a `CollateralConfigurationModule`?  can move the set max amount functions into there as well.  not urgent though, fine for now